src/main/java/core/basesyntax/Bulldozer.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+public class Bulldozer extends Machine {
+
```

Comment:
<img width="1013" alt="Screenshot 2022-01-17 at 08 58 18" src="https://user-images.githubusercontent.com/33026471/149722105-9404f1ff-9f5b-4331-9126-81a7253cb638.png">
Let's fix it in all places including main